src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -202,46 +159,157 @@ public boolean wristAtSetpoint() {
         return wristPID.atSetpoint();
     }
 
-    @Override
-    public void initSendable(SendableBuilder builder) {
-        super.initSendable(builder);
-        builder.addDoubleProperty("WristGoalPos", () -> goalPosRad[1], goal -> goalPosRad[1] = goal);
-        builder.addDoubleProperty("ArmPos", () -> getArmPos(), null);
-        builder.addDoubleProperty("WisPos", () -> getWristPos(), null);
-        builder.addDoubleProperty("RobotWisPos", () -> getWristPosRelativeToGround(), null);
-        builder.addDoubleProperty("kP: Arm", () -> kP[ARM], x -> kP[ARM] = x);
-        builder.addDoubleProperty("kP: Wis", () -> kP[WRIST], x -> kP[WRIST] = x);
-        builder.addDoubleProperty("kI: Arm", () -> kI[ARM], x -> kI[ARM] = x);
-        builder.addDoubleProperty("kI: Wis", () -> kI[WRIST], x -> kI[WRIST] = x);
-        builder.addDoubleProperty("kD: Arm", () -> kD[ARM], x -> kD[ARM] = x);
-        builder.addDoubleProperty("kD: Wis", () -> kD[WRIST], x -> kD[WRIST] = x);
-    }
-
     public boolean isCubeMode() {
-        return isCube == 0;
+        return object == CUBE;
     }
+
     public boolean isFrontMode() {
         return isFront;
     }
+
     public void toggleCube() {
-        isCube++;
-        isCube %= 2;
+        object++;
+        object %= 2;
     }
+
     public void toggleFront() {
         isFront = !isFront;
     }
```

Comment:
Rather than having all these toggle methods and an integer that stores the current state of the toggle, etc. Just implement Uday's requested OI, should be easier.

Reply:
I thought the toggles is what Uday wanted?

Reply:
[I don't think so](https://chat.carlmontrobotics.org/team-199/pl/jhxappmjwfbrdcrkehozbzpkke)

Reply:
Done.